utils/RectorGenerator/src/Command/CreateRectorCommand.php

```diff
@@ -136,16 +156,36 @@ private function processComposerAutoload(array $templateVariables): void
             return;
         }
 
-        $namespace = 'Rector\\' . $package . '\\';
-        $namespaceTest = 'Rector\\' . $package . '\\Tests\\';
+        if ($package === self::UTILS_KEYWORD) {
+            $namespace = 'Utils\\Rector\\';
+            $namespaceTest = 'Utils\\Rector\\Tests\\';
+        } else {
+            $namespace = 'Rector\\' . $package . '\\';
+            $namespaceTest = 'Rector\\' . $package . '\\Tests\\';
+        }
 
         // already autoloaded?
         if (isset($composerJson['autoload']['psr-4'][$namespace])) {
             return;
         }
 
-        $composerJson['autoload']['psr-4'][$namespace] = 'packages/' . $package . '/src';
-        $composerJson['autoload-dev']['psr-4'][$namespaceTest] = 'packages/' . $package . '/tests';
+        if ($package === self::UTILS_KEYWORD) {
+            $srcDirectory = 'utils/rector/src';
+            $testsDirectory = 'utils/rector/tests';
+        } else {
+            $srcDirectory = 'packages/' . $package . '/src';
+            $testsDirectory = 'packages/' . $package . '/tests';
+        }
+
+        // is the namespace already in composer.json? → skip it
```

Comment:
Ask user questions if they want to change their composer.json  not, mainly in Utils set